spec/services/gitlab_issues_spec.rb

```diff
@@ -0,0 +1,4 @@
+require 'spec_helper'
+
+describe AhaServices::GitlabIssues do
```

Comment:
Should add specs here for the GitLab methods -- can reference the [GitHub specs](https://github.com/aha-app/aha-services/blob/master/spec/services/github_issues_spec.rb) for an example of what's needed